README.md

```diff
@@ -89,17 +89,255 @@ Name of the application pool.
 
 The state of the ApplicationPool. Must be either `Started` or `Stopped`
 
+#### `auto_start`
+
+When true, indicates to the World Wide Web Publishing Service (W3SVC) that the application pool should be automatically started when it is created or when IIS is started. Valid options `true` or `false`
+
+#### `clr_config_file`
+
+Specifies the .NET configuration file for the application pool
+
+#### `enable32_bit_app_on_win64`
+
+When true, enables a 32-bit application to run on a computer that runs a 64-bit version of Windows. Valid options `true` or `false`
+
+#### `enable_configuration_override`
+
+When true, indicates that delegated settings in Web.config files will processed for applications within this application pool. When false, all settings in Web.config files will be ignored for this application pool. Valid options `true` or `false`
+
 #### `managed_pipeline_mode`
 
-The managedpipelinemode of the ApplicationPool. Must be either `Integrated` or `Classic`
+Specifies the request-processing mode that is used to process requests for managed content. Must be either `Integrated` or `Classic`
+
+#### `managed_runtime_loader`
+
+Specifies the managed loader to use for pre-loading the the application pool.Note: This property was added in IIS 7.5.The default value is `webengine4.dll`
 
 #### `managed_runtime_version`
 
-The managedruntimeversion of the ApplicationPool.
+Specifies the .NET Framework version that is used by the application pool.
+
+#### `pass_anonymous_token`
+
+When true, the Windows Process Activation Service (WAS) creates and passes a token for the built-in IUSR anonymous user account to the Anonymous authentication module. The Anonymous authentication module uses the token to impersonate the built-in account. When false, the token will not be passed.Note: The IUSR anonymous user account replaces the IIS_MachineName anonymous account. The IUSR account can be used by IIS or other applications. It does not have any privileges assigned to it during setup. Valid options `true` or `false`
+
+#### `start_mode`
+
+Specifies the startup type for the application pool. Valid options `:OnDemand` or `:AlwaysRunning`
+
+#### `queue_length`
+
+Indicates to HTTP.sys how many requests to queue for an application pool before rejecting future requests. When the value set for this property is exceeded, IIS rejects subsequent requests with a 503 error. If the `load_balancer_capabilities` setting is true, the connection is closed instead of rejecting requests with a 503. For more information about `load_balancer_capabilities`, see https://www.iis.net/configreference/system.applicationhost/applicationPools/add/failure". Valid options 11 to 65535.
+
+
+#### `cpu_action`
+
+Configures the action that IIS takes when a worker process exceeds its configured CPU limit. Valid options `:NoAction`, `:KillW3wp`, `:Throttle` or `:ThrottleUnderLoad`.
```

Comment:
Again, if the values are passed as strings, should be in single quotes rather than backticks.